vllm_gaudi/ops/hpu_compressed_tensors.py

```diff
@@ -448,8 +441,8 @@ def apply_weights(self,
         return output.reshape(out_shape)
 
 
-@CustomOp.register_oot(name='CompressedTensorsWNA16MoEMethod')
-class HPUCompressedTensorsWNA16MoEMethod(CompressedTensorsWNA16MoEMethod):
+@CustomOp.register_oot(name='CompressedTensorsWNA16MarlinMoEMethod')
+class HPUCompressedTensorsWNA16MoEMethod(CompressedTensorsWNA16MarlinMoEMethod):
```

Comment:
Changed parent class to CompressedTensorsWNA16MarlinMoEMethod because MoE with g_idx is unsupported in CompressedTensorsWNA16MoEMethod